torchaudio/functional.py

```diff
@@ -844,6 +844,152 @@ def compute_deltas(specgram, win_length=5, mode="replicate"):
 
 
 @torch.jit.script
+def gain(waveform, gain_db=1.0):
+    # type: (Tensor, float) -> Tensor
+    r"""Apply amplification or attenuation to the audio signal.
+
+    Args:
+       waveform (torch.Tensor): Tensor of audio of dimension (channel, time)
+       gain_db (float) Gain adjustment in decibels (dB) (Default: `1.0`).
+
+    Returns:
+       torch.Tensor: waveform amplified by gain_db
+    """
+    if (gain_db == 0):
+        return waveform
+
+    ratio = 10 ** (gain_db / 20)
+
+    return waveform * ratio
+
+
+@torch.jit.script
+def scale_to_interval(waveform, interval=1.0):
+    # type: (Tensor, float) -> Tensor
+    r"""Scales waveform to an interval.
+    Args:
+       waveform (torch.Tensor): Tensor of audio of dimension (channel, time)
+       interval (float): The bounds of the interval, where the float indicates
+           the upper bound and the negative of the float indicates the lower
+           bound (Default: `1.0`)
+           Example: interval=1.0 -> [-1.0, 1.0]
+    Returns:
+       torch.Tensor: waveform scaled to interval
+    """
+    abs_max = torch.max(torch.abs(waveform))
```

Comment:
Let's add a note in the documentation to say that this normalize the "whole waveform" ? (as opposed to "per channel", or "per batch").